### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by alisonjo315** [pantheon-systems/drops-8-scaffolding#2]
It would be cool to have a little bit of documentation about how this repo/scaffolding source works when you use it **the way it's used in in example-drops-8-composer**.:sparkle:

AND/OR, could provide de facto documentation by pointing to where this scaffolding source is "called" [in ED8C composer.json](https://github.com/pantheon-systems/example-drops-8-composer/blob/b59c7ce703b079f42149f80318ffce315d8fd3dd/composer.json#L95) and then just link off to the configuration instructions provided by `drupal-composer/drupal-scaffold`, like "[more info on this configuration and other options is over here](https://github.com/drupal-composer/drupal-scaffold#configuration)" (or a better sentence than that).

.........................................
**:sparkle: "...the way it's used in ED8C,"** as far as I can tell:
1. require drupal-composer/drupal-scaffold in your project
1. various mentions in `scripts` section to tell composer when you want it to grab scaffold files
1. set drupal-scaffold `source` and includes/excludes options in `extras` section
* **outcome:** you'll have the default drupal-scaffold files EXCEPT the excluded ones, PLUS the additional files included from this drops-8-scaffolding source)

.........................................

I was originally thinking I'd submit a PR, but then I saw a comment indicating this repo is generated by a script, soooo maybe this request isn't actually practical, or...?  Alternatively, I can add this issue to [example-drops-8-composer](https://github.com/pantheon-systems/example-drops-8-composer) -- just lmk!

Thank you!

**Comment by greg-1-anderson** [comment]
I am thinking about reworking this repo (and example-drops-8-composer) to function along the lines of what is being built in the [Composer in Core Initiative](https://www.drupal.org/about/strategic-initiatives/composer). This is blocked on https://www.drupal.org/project/drupal/issues/2982684, but hopefully that will be merged soon. If you click around, you will find some prototype example projects (like [greg-1-anderson/drupal-drupal-composer](https://github.com/greg-1-anderson/drupal-drupal-composer)) that show the general idea.

**Comment by alisonjo315** [comment]
Saw your reply in June, didn't realize I didn't reply but basically, "cool, gotcha" -- and, https://www.drupal.org/project/drupal/issues/2982684 was indeed merged in a couple-few weeks after your comment.  (And now I'm reading [the (seriously awesome) guide](https://www.drupal.org/docs/develop/using-composer/using-drupals-composer-scaffold).)

Maybe a new issue for updating this repo to line up with the new `composer-scaffold` plugin? -- is that still what you're thinking, or?

...............
Re: documentation: if this repo is generated by a script, is it actually ok for the documentation to live in this project's README, or should it be in ED8C, or?

**Comment by greg-1-anderson** [comment]
This project will be retired completely at some point, hopefully soon.

**Comment by greg-1-anderson** [comment]
The README on the master branch of this repo is the README from Drupal, so we do not have a good place for documentation. It probably would have been forward thinking to have the master branch of this repo empty except for documentation, but at this point I don't know if anyone is depending on `master`.

For anyone looking for documentation, I will point you towards [this comment](https://github.com/pantheon-systems/drops-8-scaffolding/issues/10#issuecomment-632176917) on upgrading to drupal/core-composer-scaffold.
